pynvim/api/nvim.py

```diff
@@ -579,3 +579,7 @@ def __call__(self, *args, **kwargs):
 
 class NvimError(Exception):
     pass
+
+
+class NvimKeyError(KeyError, NvimError):
+    pass
```

Comment:
Should we move errors to `nvim.api.errors` or similar?  This would likely aboid circular imports then (as mentiones in `transform_keyerror`).